Roman Bychkov/Lesson24/CalendarApp/CalindarTest/UnitTest1.cs

```diff
@@ -0,0 +1,105 @@
+using CalendarApp.BLL.Services;
+using CalendarApp.Contracts.Models;
+using CalendarApp.DAL.Repositories.Interfaces;
+using Moq;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace CalindarTest
+{
+    public class UnitTest1
```

Comment:
Name of test class should correspond to class that you test (ex. RoomServiceTests, ...)